.github/workflows/release.yml

```diff
@@ -2,12 +2,8 @@ name: Changelog
 
 on:
   push:
-    paths-ignore:
-      - '.vscode/**'
-      - 'assets/**'
-      - 'docs/**'
-      - 'examples/**'
-      - 'www/**'
+    paths:
+      - '.changeset/**'
```

Comment:
this should only run when `.changeset` directory changes, which is the only time that the release PR should change